fms_fsdp/utils/train_utils.py

```diff
@@ -132,21 +132,26 @@ def train(
                 print("gradient norm:", current_gnorm)
                 print(
                     f"speed for these {cfg.report_interval} steps:",
-                    (time.time() - start) / cfg.report_interval,
+                    current_elapsed_time,
```

Comment:
label is "speed" but variable name is "elapsed time" which doesn't seem to describe a rate. I think one or the other (or maybe both) isn't quite clear?
it might help to add units too. I think what's actually being printed is maybe time per step?

Reply:
@nairbv fixed.